src/engine/commands/migration/commands/commit.ts

```diff
@@ -40,10 +43,26 @@ export default {
         ? await uploadProject(context, options)
         : { buildName: null };
 
+      let migrationNames: string[] = [];
```

Comment:
this functionality should works only for modes `CommitMode.FULL and  ONLY_MIGRATIONS'